FLEDGE.md

```diff
@@ -1136,13 +1146,9 @@ Note that the key fields are used by the browser both to verify the signature, a
 
 The browser ensures, using TLS, the authenticity and integrity of information provided to the auction through calls made directly to an ad tech's servers. This guarantee is not provided for data passed in `runAdAuction()`. To account for this, additional bids use the same HTTP response header interception mechanism that's already in use for the [Bidding & Auction response blob](FLEDGE_browser_bidding_and_auction_API.md#step-3-get-response-blobs-to-browser) and `directFromSellerSignals`.
 
-To use HTTP response headers to convey the additional bids, the request to fetch them will first need to specify the `adAuctionHeaders` fetch flag.
-
-```
-fetch("https://...", {adAuctionHeaders: true});
-```
+To use HTTP response headers to convey the additional bids, the request to fetch them will first need to specify the `adAuctionHeaders` Fetch flag or `adAuctionHeaders` attribute on an iframe element. The procedure for specifying `adAuctionHeaders` on a Fetch or an iframe navigation is described in section [2.5.2 Using Response Headers](#252-using-response-headers), except that the response would use `Ad-Auction-Additional-Bid` response headers instead of `Ad-Auction-Signals`, and the caller of `runAdAuction()` would specify `additionalBids` on the auction config instead of or in addition to `directFromSellerSignalsHeaderAdSlot`.
 
-This signals to the browser that it should look for one or more additional bids encoded as HTTP response headers from this Fetch. Each instance of the `Ad-Auction-Additional-Bid` response header will correspond to a single additional bid. The response may include more than one additional bid by specifying multiple instances of the `Ad-Auction-Additional-Bid` response header. The structure of each instance of the `Ad-Auction-Additional-Bid` header must be as follows:
+Using `adAuctionHeaders` will cause the Fetch or navigation request to include an additional request header, `Sec-Ad-Auction-Fetch: ?1`. This also signals to the browser that it should look for one or more additional bids encoded as HTTP response headers from this Fetch or navigation. Each instance of the `Ad-Auction-Additional-Bid` response header will correspond to a single additional bid. The response may include more than one additional bid by specifying multiple instances of the `Ad-Auction-Additional-Bid` response header. The structure of each instance of the `Ad-Auction-Additional-Bid` header must be as follows:
```

Comment:
There is something i am not understand and would appreciate covered here (or perhaps one could point me to where it is covered):

If there are four slots on the page and the page solicits auction config from a seller once for all four slots and there are four additional bids, one for each slot, how does the browser sort out which is for which slot? Is there some sort of slot id on the additional bid response?